tracer/test/Datadog.Trace.Tests/Propagators/MultiSpanContextPropagatorTests.cs

```diff
@@ -42,7 +47,63 @@ static MultiSpanContextPropagatorTests()
                             ContextPropagationHeaderStyle.B3SingleHeader,
                         };
 
-            Propagator = SpanContextPropagatorFactory.GetSpanContextPropagator(names, names);
+            Propagator = SpanContextPropagatorFactory.GetSpanContextPropagator(names, names, true);
+
+            // W3CTraceContext-Datadog Extracts first header only
+            W3CDatadogPropagatorExtractFirstTrue = SpanContextPropagatorFactory.GetSpanContextPropagator(
+    new[]
```

Comment:
nit: I think some of this whitespace got messed up. For readability it should be indented like the other calls